AdobeStockAdminUi/Model/System/Config/Comment.php

```diff
@@ -89,6 +89,12 @@ private function getRedirectUrl(): string
      */
     private function getRedirectUrlPattern(): string
     {
-        return $this->getRedirectUrl();
+        $url = $this->getRedirectUrl();
+
+        /* Escape dots in the hostname */
+        $hostName = parse_url($url, PHP_URL_HOST);
+        $escapedHostName = str_replace('.', '\\\.', $hostName);
```

Comment:
I think all periods have to be escaped with `\\`, so there is a need to extract hostname before replacing. Also:
```suggestion
        $escapedHostName = str_replace('.', '\\.', $hostName);
```

Reply:
Basically, the console allows periods in other parts of the pattern (disallowed only for the hostname). But I believe if we escape everywhere, it should still work. I will double-check this point.
As for the suggestion, we need three backslashes there since we need to make the string processor know that one of our backslashes it's exact a backslash but not the escape character. I.e.
```
'that\'s a string`
`that\'s a string with the backslash -> \\'
```

Reply:
You are right. Yes, it's interesting how the string processor works, because both of backslashes are expected to be exactly backslashes in the result string. Still only one of them needs escaping.